.changeset/wet-geckos-check.md

```diff
@@ -0,0 +1,9 @@
+---
+"@snapwp/e2e-tests": patch
+"@snapwp/blocks": patch
+"@snapwp/query": patch
+"@snapwp/core": patch
+"@snapwp/next": patch
+---
+
+lint: Enforced parameter type in function documentation by adding the `jsdoc/require-param-type` ESLint rule.
```

Comment:
@justlevine We might need to add rules for this in the `Contributing.md`

Reply:
1. It's here https://github.com/rtCamp/snapwp/blob/develop/.github/CONTRIBUTING.md

2. IMO - If this is flagged by our linters than we don't need an explicit rule beyond whats in https://github.com/rtCamp/snapwp/blob/develop/DEVELOPMENT.md where we say we enforce ESLint and your PRs need to pass.